src/MCPClient/lib/clientScripts/create_mets_v2.py

```diff
@@ -66,8 +66,9 @@
 
 # archivematicaCommon
 from archivematicaFunctions import escape
-from archivematicaFunctions import strToUnicode
 from archivematicaFunctions import normalizeNonDcElementName
+from archivematicaFunctions import strToUnicode
+from archivematicaFunctions import unicodeToStr
```

Comment:
As the no. of imports starts to increase, we can adopt a different style, which for me is a little nicer to read, but ymmv. 
```
# archivematicaCommon
from archivematicaFunctions import (
    escape,
    normalizeNonDcElementName,
    strToUnicode,
    unicodeToStr,
)
```
But I can see the pattern adopted in the file, so it doesn't seem that important to add another line. Up to you.

Reply:
Cool. Thanks for pointing it out. I'll pay more attention.

I love that we use `black` now because it removes a lot of "taste based" decisions from our workflow. We could include `isort` in our tool chain too to handle our imports. Check its [Multi line output modes](https://github.com/timothycrosley/isort#multi-line-output-modes)

Reply:
Cole suggested the same! https://adr.archivematica.org/0004-isort-import-ordering.html

Reply:
Thanks! I didn't know about that ADR :grimacing: